modules/account-lib/src/coin/cspr/transactionBuilderFactory.ts

```diff
@@ -22,8 +23,19 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
   }
 
   /** @inheritDoc */
-  from(raw: Uint8Array | string): TransactionBuilder {
-    throw new NotImplementedError('from not implemented');
+  from(raw: DeployUtil.Deploy): TransactionBuilder {
+    this.validateRawTransaction(raw);
+    const tx = new Transaction(this._coinConfig);
+    tx.casperTx = raw;
+
+    if (tx.casperTx.session.tag === 5) {
```

Comment:
nit:  replace magic number with constant name to clarify

Reply:
Yes, thank you, we also should add a enum in order to catch future tags